miso-web/src/main/webapp/scripts/library_ajax.js

```diff
@@ -1051,7 +1051,18 @@ Library.ui = {
       alert("Please select one or more Libraries to dilute.");
       return false;
     }
-    alert("Finish methods to actually make the table and page.");
+    var cageDiv = '<div id="cageDialog"><span class="dialog">Look for this feature in the next release!<br>' 
+      + '<img src="http://images.mentalfloss.com/sites/default/files/styles/insert_main_wide_image/public/uncanny_valley.jpg"/></span></div>';
+    document.getElementById('go').insertAdjacentHTML('afterend', cageDiv);
```

Comment:
This line throws a TypeError in the javascript console. `document.getElementById(...) is null`